benchmark/utils/DriverUtils.swift

```diff
@@ -208,9 +208,6 @@ struct TestConfig {
     }
 
     if let x = benchArgs.optionalArgsMap["--sleep"] {
-      if x.isEmpty {
-        return .fail("--sleep requires a non-empty integer value")
-      }
       let v: Int? = Int(x)
       if v == nil {
```

Comment:
Good point! Couldn't you simplify this further by using a guard let? I.e.:

```
if let x = benchArgs.optionalArgsMap["--sleep"] {
  guard let v = Int(x) else {
    return .fail("...")
  }
  afterRunSleep = v
}
```

Reply:
Yes I can, thats even now better now